examples/app.py

```diff
@@ -31,10 +31,10 @@
 
     .. code-block:: console
 
-        cdvirtualenv src/invenio-marc21
-        pip install -e .[all]
-        pip install invenio-theme==1.0.0a9
-        pip install invenio-assets==1.0.0a4
+        $ cdvirtualenv src/invenio-marc21
+        $ pip install -e .[all]
+        $ pip install "invenio-theme>=1.0.0a9"
+        $ pip install "invenio-assets>=1.0.0a4"
```

Comment:
What about introducing an upper limit as well in case the themes or assets develop more widely than marc21 example app?  We've been bitten by this in Invenio 2.0 that is not changing much anymore... https://github.com/inveniosoftware/invenio/pull/3604